static/src/javascripts/projects/commercial/modules/prebid/prebid.js

```diff
@@ -50,115 +118,123 @@ class PrebidAdUnit {
     }
 }
 
-class PrebidService {
-    static initialise(): void {
-        const userSync = config.get('switches.prebidUserSync', false)
-            ? {
-                  // syncsPerBidder: 0, // allow all syncs - bug https://github.com/prebid/Prebid.js/issues/2781
-                  syncsPerBidder: 999, // temporarily until above bug fixed
-                  filterSettings: {
-                      all: {
-                          bidders: '*', // allow all bidders to sync by iframe or image beacons
-                          filter: 'include',
-                      },
+let requestQueue: Promise<void> = Promise.resolve();
+
+const initialise = (window: {
+    pbjs: {
+        setConfig: PbjsConfig => void,
+        bidderSettings: BidderSettings,
+        enableAnalytics: ([EnableAnalyticsConfig]) => void,
```

Comment:
This object on the window could have it's own standalone type, but I decided to leave it inline. Thoughts @kelvin-chappell @katebee ?

Reply:
Best inline I think.